ecl/hql/hqlutil.cpp

```diff
@@ -8855,9 +8859,9 @@ bool ConstantRowCreator::processFieldValue(IHqlExpression * optLhs, ITypeInfo *
         {
             assertex(optLhs);
             IHqlExpression * field = optLhs->queryChild(1);
-            if (!field->hasAttribute(countAtom) && !field->hasAttribute(sizeofAtom))
+            if (!field || (!field->hasAttribute(countAtom) && !field->hasAttribute(sizeofAtom)))
```

Comment:
This test doesn't seem right.  I think the problem is that the function is called inconsistently - sometimes it is passed a no_field, and sometimes a no_select.  I suspect it should always be a no_field.
I would add an assert that this was true, and then change processElement() to pass the field instead of the select.